packages/dappmanager/src/calls/avahi.ts

```diff
@@ -0,0 +1,36 @@
+import { HttpsLocalProxyingStatus, LocalNetworkState } from "../types";
+import { avahiController } from "../daemons/avahi";
+import {
+  httpsLocalProxyingEnableDisable,
+  httpsLocalProxyingGet
+} from "./httpsPortal";
+
+/** AVAHI: enable/disable avahi daemon consists in 2 steps:
+ * 1. Change HTTPs LOCAL_PROXYING variable
+ * 2. Start/stop avahi dameon
+ */
+export async function avahiEnableDisable(enable: boolean): Promise<void> {
```

Comment:
Rename all this routes and file to local network or something similar to stay consistent everywhere